charts/k8s-monitoring/values.yaml

```diff
@@ -1297,18 +1291,16 @@ grafana-agent-logs:
       - effect: NoSchedule
         operator: Exists
 
-  # Skip installation of the Grafana Agent CRDs, since we don't use them in this chart
+  # Skip installation of the Grafana Alloy CRDs, since we don't use them in this chart
   crds: {create: false}
 
 
-# Settings for the Grafana Agent deployment that gathers profiles.
-# You can use this sections to make modifications to the Grafana Agent deployment.
-# See https://github.com/grafana/agent/tree/main/operations/helm/charts/grafana-agent for available values.
+# Settings for the Grafana Alloy instance that gathers profiles.
+# See https://github.com/grafana/alloy/tree/main/operations/helm/charts/alloy for available values.
 # @ignored -- This skips including these values in README.md
-grafana-agent-profiles:
-  agent:
-    # This chart is creating the configuration, so the grafana-agent chart does
-    # not need to.
+alloy-profiles:
```

Comment:
All of the profiling components are public preview, which means that for them to work you _must_ configure the `--stability.level` flag to `public-preview` or the new `alloy.stabilityLevel` field in the Alloy Helm chart to `public-preview`. 

If the chart uses any other non-GA components by default you'll want to ensure stabilityLevel is set correctly there as well.

Reply:
I'll check for the other ones, but in the meantime, I've set stabilityLevel to "public-preview" for the Alloy instance that gathers and delivers profiles.